components/public-api/gitpod/v1/workspace.proto

```diff
@@ -700,3 +739,17 @@ message ParseContextURLResponse {
   WorkspaceMetadata metadata = 1;
   WorkspaceSpec spec = 2;
 }
+
+message WorkspaceClass {
+  // id is the unique identifier of the workspace class
+  string id = 1;
```

Comment:
I wonder in spec we have just class, should it be class_id then? @csweichel wdyt?